### ISSUE TITLE
polychoric matrix / Matrices policóricas 

### ISSUE THREAD

**Issue by AlejandroPeru** [jasp-stats/jasp-issues#893]
Sobre las matrices policóricas, sugiero que las puedan implementar, actualmente hay una tendencia en psicometría, en especial el  análisis factorial y la estimación de coefcinientes de confiabilidad, debido a que estas matrices policóricas/tetracóricas toman en cuenta la ordinalidad del ítem en relación al nivel de medición continuo de la variable latente. Sería interesante que revisaran el trabajo que se está haciendo alrededor de la implementación del tema, de ahí se obtiene un alfa ordinal, omega ordinal. Les invito a revisar el trabajo de los programadores de FACTOR http://psico.fcep.urv.es/utilitats/factor/Download.html .

**Comment by EJWagenmakers** [comment]
Automatic translate (of interest to @vandenman , @juliuspf  @JohnnyDoorn):
"Regarding polychoric matrices, I suggest that they can be implemented, currently there is a trend in psychometry, especially factor analysis and estimation of reliability coefficients, because these polychoric / tetrachoric matrices take into account the ordinality of the item in relation to the level of continuous measurement of the latent variable. It would be interesting for them to review the work that is being done around the implementation of the topic, hence an ordinal alpha is obtained, omega ordinal. I invite you to review the work of the FACTOR programmers http://psico.fcep.urv.es/utilitats/factor/Download.html ."

E.J.

**Comment by AlejandroPeru** [comment]
@e

**Comment by AlejandroPeru** [comment]
http://www.scielo.edu.uy/scielo.php?script=sci_arttext&pid=S1688-42212013000200005  @EJWagenmakers

**Comment by ivmcphail** [comment]
I want to second this request. If the polychoric/tetrachoric approach to correlations and Exploratory Factor Analysis and Confirmatory Factor Analysis (and the various analyses within EFA/CFA, like polychoric parallel analysis) could be added to JASP Factor Analysis module, that would be very excellent! I use these features in R a lot and will be doing so a lot in the future, and having a faster way to do these analyses and creating the various graphs, this would be a dream!

**Comment by mike-ology** [comment]
+1 to the request. 

I actually made a similar request over a year ago that was closed but didn't seem to result in any feature additions (#379). It would be nice to have some update on the status of this feature as I have seen it requested multiple times now. I do a lot of work using ordinal data (factor analyses of questionnaire responses) and it is a shame that JASP isn't fit for handling these analyses yet :(
